docs/multi-stage-query/reference.md

```diff
@@ -45,8 +45,11 @@ making it easy to reuse the same SQL statement for each ingest: just specify the
 
 ### `EXTERN` Function
 
-Use the `EXTERN` function to read external data. The function has two variations.
+Use the `EXTERN` function to read external data or write to an external source.
```

Comment:
When we are writing to an external location, it is not a source. Something like the following might be better. 
```suggestion
Use the `EXTERN` function to read external data or write to an external location.
```